pages/EventsAdmin/EventEditor.jsx

```diff
@@ -316,13 +316,17 @@ class EventEditor extends React.Component {
                         type='number' onChange={ this.onEventGameOptionChange.bind(this, 'gameTimeLimit') } value={ this.state.eventGameOptions.gameTimeLimit } />
                     }
                     { this.state.useEventGameOptions
-                    && <Checkbox name='useChessClocks' label='Use chess clocks with a time limit per player (in minutes)' labelClass='col-sm-4' fieldClass='col-sm-offset-3 col-sm-8'
+                    && <Checkbox name='useChessClocks' label='Use chess clocks with a time limit per player' labelClass='col-sm-4' fieldClass='col-sm-offset-3 col-sm-8'
```

Comment:
Similar to the lobby screen, just need to update this page to ensure that you cannot have both chess clock & regular clock ticked:
![image](https://github.com/throneteki/throneteki-client/assets/23492047/559685da-0f92-435c-83c2-f2e00c55f546)

Reply:
good catch, fixed!